khan-academy/advanced_sql_queries.sql

```diff
@@ -0,0 +1,340 @@
+    "United Arab Emirates", 9445624, 1.06, 99495, 153484, 30.7, 1, 1.85, 83600, 113, 85, 8047522, 0.13
+);
+INSERT INTO countries_by_population (country, population, percent_one_year_change, population_change, net_migrants, median_age, percent_aged_over_60, fertility_rate, area_sq_km, density_per_sq_km, urban_pop_percent, urban_pop, percent_of_world_pop) VALUES (
+    "Belarus", 9307609, -0.52, -49069, 2650, 39.4, 20, 1.47, 207602, 45, 76, 7095843, 0.13
+);
+INSERT INTO countries_by_population (country, population, percent_one_year_change, population_change, net_migrants, median_age, percent_aged_over_60, fertility_rate, area_sq_km, density_per_sq_km, urban_pop_percent, urban_pop, percent_of_world_pop) VALUES (
+    "Austria", 8526429, 0.37, 31284, 30371, 43.0, 24, 1.45, 83858, 102, 68, 5824085, 0.12
+);
+INSERT INTO countries_by_population (country, population, percent_one_year_change, population_change, net_migrants, median_age, percent_aged_over_60, fertility_rate, area_sq_km, density_per_sq_km, urban_pop_percent, urban_pop, percent_of_world_pop) VALUES (
+    "Tajikistan", 8408947, 2.45, 201113, -18197, 21.9, 5, 3.82, 143099, 59, 27, 2246516, 0.12
+);
+INSERT INTO countries_by_population (country, population, percent_one_year_change, population_change, net_migrants, median_age, percent_aged_over_60, fertility_rate, area_sq_km, density_per_sq_km, urban_pop_percent, urban_pop, percent_of_world_pop) VALUES (
+    "Honduras", 8260749, 2.01, 163061, -11487, 22.2, 7, 3.08, 112088, 74, 54, 4449281, 0.11
+);
+INSERT INTO countries_by_population (country, population, percent_one_year_change, population_change, net_migrants, median_age, percent_aged_over_60, fertility_rate, area_sq_km, density_per_sq_km, urban_pop_percent, urban_pop, percent_of_world_pop) VALUES (
+    "Switzerland", 8157896, 0.99, 80063, 64965, 42.1, 24, 1.52, 41284, 198, 74, 6030730, 0.11
+);
+INSERT INTO countries_by_population (country, population, percent_one_year_change, population_change, net_migrants, median_age, percent_aged_over_60, fertility_rate, area_sq_km, density_per_sq_km, urban_pop_percent, urban_pop, percent_of_world_pop) VALUES (
+    "Israel", 7822107, 1.15, 88963, 19636, 30.1, 15, 2.91, 22145, 353, 92, 7201448, 0.11
+);
+INSERT INTO countries_by_population (country, population, percent_one_year_change, population_change, net_migrants, median_age, percent_aged_over_60, fertility_rate, area_sq_km, density_per_sq_km, urban_pop_percent, urban_pop, percent_of_world_pop) VALUES (
+    "Jordan", 7504812, 3.18, 231013, 81911, 23.7, 5, 3.34, 89342, 84, 83, 6261026, 0.10
+);
+INSERT INTO countries_by_population (country, population, percent_one_year_change, population_change, net_migrants, median_age, percent_aged_over_60, fertility_rate, area_sq_km, density_per_sq_km, urban_pop_percent, urban_pop, percent_of_world_pop) VALUES (
+    "Papua New Guinea", 7476108, 2.12, 154846, 0, 21.0, 5, 3.86, 462831, 16, 13, 950364, 0.10
+);
+
```

Comment:
You could shorten this code down a lot by turning this into a single query rather than a bunch of little ones doing the same thing. So, for example doing something like,
```
NSERT INTO countries_by_population (country, population, percent_one_year_change, population_change, net_migrants, median_age, percent_aged_over_60, fertility_rate, area_sq_km, density_per_sq_km, urban_pop_percent, urban_pop, percent_of_world_pop) VALUES (
(China, bla, blah blah, blah, blah),
(Murica, blah blah blah, blah, blah, blah)
);
```